src/pause/pause_badges.c

```diff
@@ -483,7 +487,11 @@ void pause_badges_draw_contents(MenuPanel* menu, s32 baseX, s32 baseY, s32 width
                                         badgeListY + pause_badges_scroll_offset_y(posY) + badgeEntryOffsetY, 255, palette, style);
                             }
                         }
+#if VERSION_JP
+                        draw_number(gMoveTable[gItemTable[badgeID].moveID].costBP, baseX + 211 + pause_badges_scroll_offset_x(posX),
```

Comment:
211 is used on JP in place of 235 on other versions, so we can make a define for this too. It can be used here and elsewhere